package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "repository": "https://github.com/artsy/metaphysics",
   "engines": {
-    "node": "12",
+    "node": "14",
```

Comment:
What do you think about changing this to `>=12`? I feel like this would make the transition easier until all the node projects are on v14.

Reply:
```suggestion
    "node": ">=12",
```